src/Inkluzitron/Modules/BdsmTestOrg/GraphPainter.cs

```diff
@@ -0,0 +1,272 @@
+﻿using DC = Discord;
+using Inkluzitron.Data.Entities;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Globalization;
+
+namespace Inkluzitron.Modules.BdsmTestOrg
+{
+    public class GraphPainter
+    {
+        public Color BackgroundColor { get; set; } = Color.Black;
+        public int CategoryBoxPadding { get; set; } = 20;
+        public int CategoryBoxHeight { get; set; } = 300;
+
+        public Font GridLinePercentageFont { get; set; } = new Font(SystemFonts.DefaultFont.FontFamily, 20);
+        public Brush GridLinePercentageForegroundMajor { get; set; } = new SolidBrush(Color.FromArgb(0x70AAAAAA));
+        public Brush GridLinePercentageBackgroundMinor { get; set; } = new SolidBrush(Color.FromArgb(0x32AAAAAA));
+        public Pen GridLinePenMinor { get; set; } = new Pen(Color.FromArgb(0x32AAAAAA));
+        public Pen GridLinePenMajor { get; set; } = new Pen(Color.FromArgb(0x70AAAAAA));
+
+        public Font CategoryBoxHeadingFont { get; set; } = new Font(SystemFonts.DefaultFont.FontFamily, 20);
+        public Brush CategoryBoxBackground { get; set; } = new SolidBrush(Color.FromArgb(0x7F333333));
+        public Brush CategoryBoxHeadingForeground { get; set; } = new SolidBrush(Color.FromArgb(0x7FEEEEEE));
+
+        public int AvatarSize { get; set; } = 64;
+        public Font UsernameFont { get; set; } = new Font(SystemFonts.DefaultFont.FontFamily, 20);
+        public Brush UsernameForeground { get; set; } = new SolidBrush(Color.FromArgb(0x7FFFFFDD));
+
+        public int ColumnCount { get; set; } = 5;
+
+        public async Task<Bitmap> DrawAsync(DC.WebSocket.DiscordSocketClient client, IDictionary<string, List<QuizDoubleItem>> toplistResults, float reportingThreshold)
+        {
+            async Task<Image> GetAvatarAsync(ulong userId)
```

Comment:
Zanořeným metodám bych se s velkým obloukem vyhnul. Nepůsobí to moc friendly k čitelnosti. Minimálně bych z toho udělal běžnou metodu. Ideálně nějakou helper/extension metodu (podle toho, jak je to vhodné).